.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules/*
-/build_artifacts/*
\ No newline at end of file
+/build_artifacts/*
+/test_output/*
```

Comment:
Add a new line at the end of this file. Apparently this can cause an issue for some programs, which is why github highlights it.